pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -135,9 +134,7 @@ func (s *nodeTopoInformer) Start(stopCh <-chan struct{}) {
 }
 
 func (s *nodeTopoInformer) HasSynced() bool {
-	if !s.initialTopology {
-		return false
-	}
+	klog.V(4).Infof("nodeTopoInformer need collector to start")
```

Comment:
how about adding a comment instead of a log

Reply:
V5 xxx ready to start
// TODO only node cpu info collector relies on node topo informer